src/server/endpoints/sensors.py

```diff
@@ -27,8 +27,8 @@
 PHP_INT_MAX = 2147483647
 
 
-def _authenticate(names: List[str]):
-    auth_tokens_presented = (resolve_auth_token() or "").split(",")
+def _authenticate(r: Request, names: List[str]):
+    auth_tokens_presented = (resolve_auth_token(r) or "").split(",")
```

Comment:
```suggestion
def _authenticate(req: Request, names: List[str]):
    auth_tokens_presented = (resolve_auth_token(req) or "").split(",")
```
how about `req` instead of `r`?  single-character variable names can be problematic outside of a few special cases